src/bio/bio_xstream.c

```diff
@@ -1617,13 +1702,14 @@ bio_xsctxt_alloc(struct bio_xs_context **pctxt, int tgt_id, bool self_polling)
 
 		/* Initialize all registered subsystems: bdev, vmd, copy. */
 		common_prep_arg(&cp_arg);
-		spdk_subsystem_init_from_json_config(nvme_glb.bd_nvme_conf,
-						     SPDK_DEFAULT_RPC_ADDR,
```

Comment:
Will the same default RPC address be used with these changes?

Reply:
yes, https://github.com/spdk/spdk/blob/d0bb354293fb373e21001c4fee100545d2d89744/lib/init/json_config.c#L622